Tests/Recurly/Base_Test.php

```diff
@@ -4,10 +4,24 @@
 
 class Recurly_BaseTest extends Recurly_TestCase {
 
-  public function testParsingXMLToNewObject() {
-    $this->client->addResponse('GET', '/accounts/abcdef1234567890', 'accounts/empty.xml');
+  public function testParsingEmptyXML() {
+    $this->client->addResponse('GET', 'abcdef1234567890', 'accounts/empty.xml');
 
-    $account = Recurly_Account::get('abcdef1234567890', $this->client);
+    $account = Recurly_Base::_get('abcdef1234567890', $this->client);
```

Comment:
The changes to this test are unrelated, I just realized we could avoid the child class and simplify it.